memory/libhugetlbfs.py

```diff
@@ -83,11 +83,11 @@ def setUp(self):
                                                       verbose=False,
                                                       shell=True)
             if 'HugePages_' not in Hugepages_support:
-                self.error("No Hugepages Configured")
+                self.cancle("No Hugepages Configured")
```

Comment:
typo: `cancel`. And please correct in commit message too.

Reply:
ah sure my mistake :(

Reply:
@PraveenPenguin typo needs to be changed in commit message as well.